api/tasks.py

```diff
@@ -11,3 +13,22 @@ def create_delivery(order):
     else:
         delivery.print_pdf()
         return {'success': 'all good'}
+
+
+@app.task
+def check_customer_is_partner(customer, created=True):
+    crm_client = CrmClient()
+    try:
+        is_partner = crm_client.check_customer_is_partner(customer)
+    except CrmError:
+        return {"error": "Error connecting to CRM", "customer": customer}
```

Comment:
IMO it'll be more changeable to make pick the error message from the exception itself. This way we could introduce new exception classes or change their message without having to touch this handler.